### PR TITLE
Map POC and News page

### PR DESCRIPTION
**Pull request by asotog**
requests limited to localhost and app domain

image

### CODE REVIEW

**Comment by asotog** [comment]
updated to mapbox now looks like 
image

File: src/pages/index.tsx [hunk 1]
```diff
@@ -34,6 +35,16 @@ const HomePage: NextPage<HomePageProps> = ({ meta, allCases, recentCases }) => {
             deadCases={deadCases.length}
           />
         </Section>
+        <Section title="Mapa">
```
Comment (iamkevinwolf) [inline src/pages/index.tsx]: I'm wondering if `Casos por región` makes more sense 🤔. @credondocr thoughts?

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { AppProps } from 'next/app'
 import router from 'next/router'
-
+import 'mapbox-gl/dist/mapbox-gl.css'
```
[the remark this answers is not in the document]
Comment (asotog) [reply src/pages/_app.tsx]: Nextjs does not let me import CSS from components, only from app, it throws an error

[the remark this answers is not in the document]
Comment (iamkevinwolf) [reply src/pages/_app.tsx]: Understood.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -34,6 +35,16 @@ const HomePage: NextPage<HomePageProps> = ({ meta, allCases, recentCases }) => {
...
+          <CasesMap
+            // temp data while get CMS/graphql updated
+            sampleData={[
+              { ...recentCases[0], lat: 10.0274898, lng: -84.2224415 },
```
Comment (asotog) [inline src/pages/index.tsx]: this prop needs to go away once lat and lng start to come from cases graphql query response

**Comment by iamkevinwolf** [comment]
@asotog is there a way to also add the card and table-like border-radius and box-shadow to the map container?

**Comment by asotog** [comment]
> @asotog is there a way to also add the card and table-like border-radius and box-shadow to the map container?

it is true, is missing, will add it

**Pull request closed without merging**